**Windows** [Build 4701](http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=95080&buildTypeId=WF_PullRequest_Windows) outcome was **FAILURE** using a merge of ab83148008fc0282af23536f7a0895a4030f4f04
Summary: Tests failed: 7 (7 new), passed: 3311, ignored: 194 Build time: 01:01:20
### Failed tests
```
org.jboss.as.test.clustering.cluster.web.CoarseWebFailoverTestCase(SYNC-tcp).testGracefulSimpleFailover: java.lang.AssertionError: expected:<200> but was:<500>
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:69)
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:58)
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:52)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.establishTopology(ClusteredWebFailoverAbstractCase.java:256)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.testFailover(ClusteredWebFailoverAbstractCase.java:171)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.testGracefulSimpleFailover(ClusteredWebFailoverAbstractCase.java:83)
------- Stdout: -------
00:57:47,509 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:57:47,509 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:57:47,509 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-0
00:57:47,504 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0002: Content removed from location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-1\standalone\data\content\91\bbd7c669babafb0eea43ab1dee74dc406545c6\content
00:57:47,504 INFO  [org.jboss.as.server] (management-handler-thread - 2) WFLYSRV0009: Undeployed "coarse-distributable.war" (runtime-name: "coarse-distributable.war")
00:57:47,520 INFO  [org.jboss.as.repository] (management-handler-thread - 4) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\91\bbd7c669babafb0eea43ab1dee74dc406545c6\content
00:57:47,522 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-7) WFLYSRV0027: Starting deployment of "coarse-distributable.war" (runtime-name: "coarse-distributable.war")
00:57:47,548 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.coarse-distributable.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.
00:57:47,548 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.coarse-distributable.war" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:57:47,548 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.coarse-distributable.war" is using a private module ("org.infinispan:main") which may be changed or removed in future versions without notice.
00:57:47,553 INFO  [org.jboss.weld.deployer] (MSC service thread 1-2) WFLYWELD0003: Processing weld deployment coarse-distributable.war
00:57:47,559 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-2) WFLYEJB0473: JNDI bindings for session bean named 'TopologyChangeListenerBean' in deployment unit 'deployment "coarse-distributable.war"' are as follows:

	java:global/coarse-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:app/coarse-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:module/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:jboss/exported/coarse-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:global/coarse-distributable/TopologyChangeListenerBean
	java:app/coarse-distributable/TopologyChangeListenerBean
	java:module/TopologyChangeListenerBean

00:57:47,585 INFO  [org.jboss.weld.deployer] (MSC service thread 1-5) WFLYWELD0006: Starting Services for CDI deployment: coarse-distributable.war
00:57:47,602 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,602 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,604 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 8) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,604 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 8) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,604 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 32) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,605 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 32) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,609 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 8) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,619 INFO  [org.jboss.weld.deployer] (MSC service thread 1-5) WFLYWELD0009: Starting weld service for deployment coarse-distributable.war
00:57:47,623 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 8) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,626 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 20) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,627 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 20) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:57:47,626 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 32) WFLYCLINF0002: Started client-mappings cache from ejb container
00:57:47,648 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 8) WFLYCLINF0002: Started routing cache from web container
00:57:47,649 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 32) WFLYCLINF0002: Started coarse-distributable.war cache from web container
00:57:47,656 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 35) WFLYCLINF0002: Started dist cache from ejb container
00:57:47,795 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 35) WFLYUT0021: Registered web context: /coarse-distributable
00:57:47,807 INFO  [org.jboss.as.server] (management-handler-thread - 4) WFLYSRV0010: Deployed "coarse-distributable.war" (runtime-name : "coarse-distributable.war")
00:57:47,815 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-1
00:57:47,825 INFO  [org.jboss.as.repository] (management-handler-thread - 3) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-1\standalone\data\content\91\bbd7c669babafb0eea43ab1dee74dc406545c6\content
00:57:47,829 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-7) WFLYSRV0027: Starting deployment of "coarse-distributable.war" (runtime-name: "coarse-distributable.war")
00:57:47,852 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-7) WFLYSRV0018: Deployment "deployment.coarse-distributable.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.


org.jboss.as.test.clustering.cluster.web.context.InvalidateConversationTestCase(SYNC-tcp).testInvalidate: java.lang.AssertionError: expected:<200> but was:<500>
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:69)
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:58)
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:52)
	at org.jboss.as.test.clustering.cluster.web.context.InvalidateConversationTestCase.establishTopology(InvalidateConversationTestCase.java:129)
	at org.jboss.as.test.clustering.cluster.web.context.InvalidateConversationTestCase.testInvalidate(InvalidateConversationTestCase.java:93)
------- Stdout: -------
00:58:13,497 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:58:13,498 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:58:13,498 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-0
00:58:13,499 INFO  [org.jboss.as.arquillian.protocol.jmx.ArquillianServiceDeployer] (main) Deploy arquillian service: arquillian-service: 981 assets
00:58:14,228 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\57\960110e5968e2e1ff5ab7486da4cf0c76c2f91\content
00:58:14,230 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-7) WFLYSRV0027: Starting deployment of "arquillian-service" (runtime-name: "arquillian-service")
00:58:14,378 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-1) WFLYSRV0018: Deployment "deployment.arquillian-service" is using a private module ("org.jboss.as.jmx:main") which may be changed or removed in future versions without notice.
00:58:14,378 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-1) WFLYSRV0018: Deployment "deployment.arquillian-service" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:58:14,378 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-1) WFLYSRV0018: Deployment "deployment.arquillian-service" is using a private module ("org.jboss.jandex:main") which may be changed or removed in future versions without notice.
00:58:14,378 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-1) WFLYSRV0018: Deployment "deployment.arquillian-service" is using a private module ("org.wildfly.security.manager:main") which may be changed or removed in future versions without notice.
00:58:14,385 WARN  [org.jboss.weld.deployer] (MSC service thread 1-1) WFLYWELD0013: Deployment deployment "arquillian-service" contains CDI annotations but no bean archive was not found. (No beans.xml nor class with bean defining annotations)
00:58:14,406 INFO  [org.jboss.as.server] (management-handler-thread - 2) WFLYSRV0010: Deployed "arquillian-service" (runtime-name : "arquillian-service")
00:58:14,484 WARN  [org.jboss.remotingjmx.Util] (main) The protocol 'http-remoting-jmx' is deprecated, instead you should use 'remote+http'.
00:58:14,547 INFO  [org.jboss.as.repository] (management-handler-thread - 3) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\ee\50a8835708a5ce0336a99b2339ba39d40b9e00\content
00:58:14,550 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-6) WFLYSRV0027: Starting deployment of "conversation.war" (runtime-name: "conversation.war")
00:58:14,581 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-1) WFLYSRV0018: Deployment "deployment.conversation.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.
00:58:14,581 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-1) WFLYSRV0018: Deployment "deployment.conversation.war" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:58:14,581 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-1) WFLYSRV0018: Deployment "deployment.conversation.war" is using a private module ("org.infinispan:main") which may be changed or removed in future versions without notice.
00:58:14,581 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-1) WFLYSRV0018: Deployment "deployment.conversation.war" is using a private module ("org.wildfly.security.manager:main") which may be changed or removed in future versions without notice.
00:58:14,588 INFO  [org.jboss.weld.deployer] (MSC service thread 1-1) WFLYWELD0003: Processing weld deployment conversation.war
00:58:14,593 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-1) WFLYEJB0473: JNDI bindings for session bean named 'TopologyChangeListenerBean' in deployment unit 'deployment "conversation.war"' are as follows:

	java:global/conversation/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:app/conversation/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:module/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:jboss/exported/conversation/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:global/conversation/TopologyChangeListenerBean
	java:app/conversation/TopologyChangeListenerBean
	java:module/TopologyChangeListenerBean

00:58:14,643 INFO  [org.jboss.weld.deployer] (MSC service thread 1-1) WFLYWELD0006: Starting Services for CDI deployment: conversation.war
00:58:14,648 INFO  [org.jboss.as.arquillian] (MSC service thread 1-3) Arquillian deployment detected: ArquillianConfig[service=jboss.arquillian.config."conversation.war",unit=conversation.war,tests=[org.jboss.as.test.clustering.cluster.web.context.InvalidateConversationTestCase]]
00:58:14,652 INFO  [org.jboss.weld.deployer] (MSC service thread 1-3) WFLYWELD0009: Starting weld service for deployment conversation.war
00:58:14,653 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 10) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:14,653 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 10) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:14,654 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 3) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:14,655 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 3) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:14,658 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:14,658 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:14,659 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 10) WFLYCLINF0002: Started dist cache from ejb container
00:58:14,660 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 10) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:14,661 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 10) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:14,665 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 10) WFLYCLINF0002: Started routing cache from web container
00:58:14,668 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 35) WFLYCLINF0002: Started client-mappings cache from ejb container


org.jboss.as.test.clustering.cluster.web.expiration.CoarseSessionExpirationTestCase(SYNC-tcp).test: java.lang.AssertionError: null
	at org.jboss.as.test.clustering.cluster.web.expiration.SessionExpirationTestCase.test(SessionExpirationTestCase.java:101)
------- Stdout: -------
00:58:30,426 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:58:30,426 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:58:30,427 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-0
00:58:30,437 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\cd\c51f4b5ecbbf3aafd28ba52aadd2f139bcccbe\content
00:58:30,438 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-6) WFLYSRV0027: Starting deployment of "expiration.war" (runtime-name: "expiration.war")
00:58:30,500 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 3) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,501 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 3) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,502 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 3) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,502 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 3) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,510 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 3) WFLYCLINF0002: Started routing cache from web container
00:58:30,512 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 86) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,513 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 86) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,522 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 86) WFLYCLINF0002: Started expiration.war cache from web container
00:58:30,537 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 86) WFLYUT0021: Registered web context: /expiration
00:58:30,551 INFO  [org.jboss.as.server] (management-handler-thread - 2) WFLYSRV0010: Deployed "expiration.war" (runtime-name : "expiration.war")
00:58:30,557 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-1
00:58:30,568 INFO  [org.jboss.as.repository] (management-handler-thread - 1) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-1\standalone\data\content\cd\c51f4b5ecbbf3aafd28ba52aadd2f139bcccbe\content
00:58:30,569 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0027: Starting deployment of "expiration.war" (runtime-name: "expiration.war")
00:58:30,615 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 89) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,615 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 89) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,617 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,618 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,621 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 89) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,621 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 89) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,624 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 82) WFLYCLINF0002: Started routing cache from web container
00:58:30,629 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 89) WFLYCLINF0002: Started expiration.war cache from web container
00:58:30,642 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 89) WFLYUT0021: Registered web context: /expiration
00:58:30,652 INFO  [org.jboss.as.server] (management-handler-thread - 1) WFLYSRV0010: Deployed "expiration.war" (runtime-name : "expiration.war")
00:58:30,679 INFO  [stdout] (default task-2) http://[::1]:8080/expiration/listener?operation=set&name=a;jsessionid=vEyHP3VNuWTZRS9SaRPufqgCmbWlNJZp8guPrr_7

00:58:30,700 INFO  [stdout] (default task-3) http://[::1]:8080/expiration/listener?operation=set&name=a&value=1;jsessionid=vEyHP3VNuWTZRS9SaRPufqgCmbWlNJZp8guPrr_7

00:58:30,709 INFO  [stdout] (default task-22) http://[::1]:8180/expiration/listener?operation=get&name=a;jsessionid=oh4EcdpgChmX7BA2ye_A355jwPIpcAgnim9oWlVu

00:58:30,713 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:58:30,713 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:58:30,713 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Undeploying deployment=deployment-0
00:58:30,716 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 87) WFLYUT0022: Unregistered web context: /expiration
00:58:30,723 INFO  [org.infinispan.eviction.impl.PassivationManagerImpl] (ServerService Thread Pool -- 87) ISPN000029: Passivating all entries to disk
00:58:30,742 INFO  [org.infinispan.eviction.impl.PassivationManagerImpl] (ServerService Thread Pool -- 87) ISPN000030: Passivated 3 entries in 18 milliseconds
00:58:30,745 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 87) WFLYCLINF0003: Stopped expiration.war cache from web container
00:58:30,754 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 10) WFLYCLINF0003: Stopped routing cache from web container
00:58:30,757 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-5) WFLYSRV0028: Stopped deployment expiration.war (runtime-name: expiration.war) in 40ms
00:58:30,770 INFO  [org.jboss.as.repository] (management-handler-thread - 4) WFLYDR0002: Content removed from location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\cd\c51f4b5ecbbf3aafd28ba52aadd2f139bcccbe\content
00:58:30,770 INFO  [org.jboss.as.server] (management-handler-thread - 4) WFLYSRV0009: Undeployed "expiration.war" (runtime-name: "expiration.war")
00:58:30,771 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Undeploying deployment=deployment-1
00:58:30,775 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 89) WFLYUT0022: Unregistered web context: /expiration
00:58:30,779 INFO  [org.infinispan.eviction.impl.PassivationManagerImpl] (ServerService Thread Pool -- 90) ISPN000029: Passivating all entries to disk


org.jboss.as.test.clustering.cluster.web.expiration.FineSessionExpirationTestCase(SYNC-tcp).test: java.lang.AssertionError: null
	at org.jboss.as.test.clustering.cluster.web.expiration.SessionExpirationTestCase.test(SessionExpirationTestCase.java:101)
------- Stdout: -------
00:58:30,820 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:58:30,820 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:58:30,820 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-0
00:58:30,830 INFO  [org.jboss.as.repository] (management-handler-thread - 1) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\d0\5f3170276a187340b6bfd8789e43994281a091\content
00:58:30,834 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0027: Starting deployment of "expiration.war" (runtime-name: "expiration.war")
00:58:30,888 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 87) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,889 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 87) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,889 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 87) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,890 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 87) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,915 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 87) WFLYCLINF0002: Started routing cache from web container
00:58:30,919 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,920 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:30,934 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 35) WFLYCLINF0002: Started expiration.war cache from web container
00:58:30,947 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 35) WFLYUT0021: Registered web context: /expiration
00:58:30,966 INFO  [org.jboss.as.server] (management-handler-thread - 1) WFLYSRV0010: Deployed "expiration.war" (runtime-name : "expiration.war")
00:58:30,972 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-1
00:58:30,982 INFO  [org.jboss.as.repository] (management-handler-thread - 4) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-1\standalone\data\content\d0\5f3170276a187340b6bfd8789e43994281a091\content
00:58:30,985 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-5) WFLYSRV0027: Starting deployment of "expiration.war" (runtime-name: "expiration.war")
00:58:31,047 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 89) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,048 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 89) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,048 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,048 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,051 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 85) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,051 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 85) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,058 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 85) WFLYCLINF0002: Started routing cache from web container
00:58:31,058 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 87) WFLYCLINF0002: Started expiration.war cache from web container
00:58:31,069 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 87) WFLYUT0021: Registered web context: /expiration
00:58:31,098 INFO  [org.jboss.as.server] (management-handler-thread - 4) WFLYSRV0010: Deployed "expiration.war" (runtime-name : "expiration.war")
00:58:31,125 INFO  [stdout] (default task-4) http://[::1]:8080/expiration/listener?operation=set&name=a;jsessionid=PbEJjvPGWo4LJ6uzWPb6jdXaESoJTqQglz8AXMUt

00:58:31,132 INFO  [stdout] (default task-5) http://[::1]:8080/expiration/listener?operation=set&name=a&value=1;jsessionid=PbEJjvPGWo4LJ6uzWPb6jdXaESoJTqQglz8AXMUt

00:58:31,140 INFO  [stdout] (default task-23) http://[::1]:8180/expiration/listener?operation=get&name=a;jsessionid=2Ef9LZdJPokc8-Ftr1NHzAUrzpzmzFDQLgvBRbJz

00:58:31,145 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:58:31,145 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:58:31,145 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Undeploying deployment=deployment-0
00:58:31,149 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0022: Unregistered web context: /expiration
00:58:31,153 INFO  [org.infinispan.eviction.impl.PassivationManagerImpl] (ServerService Thread Pool -- 86) ISPN000029: Passivating all entries to disk
00:58:31,156 INFO  [org.infinispan.eviction.impl.PassivationManagerImpl] (ServerService Thread Pool -- 86) ISPN000030: Passivated 3 entries in 2 milliseconds
00:58:31,158 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 86) WFLYCLINF0003: Stopped expiration.war cache from web container
00:58:31,164 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 86) WFLYCLINF0003: Stopped routing cache from web container
00:58:31,167 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment expiration.war (runtime-name: expiration.war) in 19ms
00:58:31,180 INFO  [org.jboss.as.repository] (management-handler-thread - 3) WFLYDR0002: Content removed from location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\d0\5f3170276a187340b6bfd8789e43994281a091\content
00:58:31,180 INFO  [org.jboss.as.server] (management-handler-thread - 3) WFLYSRV0009: Undeployed "expiration.war" (runtime-name: "expiration.war")
00:58:31,181 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Undeploying deployment=deployment-1
00:58:31,185 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 85) WFLYUT0022: Unregistered web context: /expiration
00:58:31,192 INFO  [org.infinispan.eviction.impl.PassivationManagerImpl] (ServerService Thread Pool -- 85) ISPN000029: Passivating all entries to disk


org.jboss.as.test.clustering.cluster.web.ExternalizerTestCase(SYNC-tcp).test: java.lang.AssertionError: expected:<3> but was:<1>
	at org.jboss.as.test.clustering.cluster.web.ExternalizerTestCase.assertValue(ExternalizerTestCase.java:109)
	at org.jboss.as.test.clustering.cluster.web.ExternalizerTestCase.test(ExternalizerTestCase.java:97)
------- Stdout: -------
00:58:31,224 INFO  [org.jboss.as.test.clustering.cluster.ClusterAbstractTestCase] (main) externalizer.war:
/WEB-INF/
/WEB-INF/classes/
/WEB-INF/classes/META-INF/
/WEB-INF/classes/META-INF/services/
/WEB-INF/classes/META-INF/services/org.wildfly.clustering.marshalling.Externalizer
/WEB-INF/classes/org/
/WEB-INF/classes/org/jboss/
/WEB-INF/classes/org/jboss/as/
/WEB-INF/classes/org/jboss/as/test/
/WEB-INF/classes/org/jboss/as/test/clustering/
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/externalizer/
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/externalizer/Counter.class
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/externalizer/CounterServlet.class
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/externalizer/CounterExternalizer.class
/WEB-INF/web.xml
00:58:31,225 INFO  [org.jboss.as.test.clustering.cluster.ClusterAbstractTestCase] (main) externalizer.war:
/WEB-INF/
/WEB-INF/classes/
/WEB-INF/classes/META-INF/
/WEB-INF/classes/META-INF/services/
/WEB-INF/classes/META-INF/services/org.wildfly.clustering.marshalling.Externalizer
/WEB-INF/classes/org/
/WEB-INF/classes/org/jboss/
/WEB-INF/classes/org/jboss/as/
/WEB-INF/classes/org/jboss/as/test/
/WEB-INF/classes/org/jboss/as/test/clustering/
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/externalizer/
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/externalizer/Counter.class
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/externalizer/CounterServlet.class
/WEB-INF/classes/org/jboss/as/test/clustering/cluster/web/externalizer/CounterExternalizer.class
/WEB-INF/web.xml
00:58:31,228 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:58:31,228 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:58:31,228 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-0
00:58:31,236 INFO  [org.jboss.as.repository] (management-handler-thread - 4) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\95\2a8dceecd71cc8d8e7f05b59f958bd944be9c8\content
00:58:31,238 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "externalizer.war" (runtime-name: "externalizer.war")
00:58:31,285 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 86) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,286 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 86) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,287 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 83) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,288 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 83) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,288 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 87) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,289 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 87) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.


org.jboss.as.test.clustering.cluster.web.GranularWebFailoverTestCase(SYNC-tcp).testGracefulUndeployFailover: java.lang.AssertionError: expected:<200> but was:<500>
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:69)
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:58)
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:52)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.establishTopology(ClusteredWebFailoverAbstractCase.java:256)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.testFailover(ClusteredWebFailoverAbstractCase.java:112)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.testGracefulUndeployFailover(ClusteredWebFailoverAbstractCase.java:103)
------- Stdout: -------
00:58:31,802 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:58:31,802 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:58:31,802 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-0
00:58:31,815 INFO  [org.jboss.as.repository] (management-handler-thread - 3) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\c7\20c52736bdb32bc8829c63904895cdfddfe625\content
00:58:31,818 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0027: Starting deployment of "granular-distributable.war" (runtime-name: "granular-distributable.war")
00:58:31,844 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.
00:58:31,844 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:58:31,844 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.infinispan:main") which may be changed or removed in future versions without notice.
00:58:31,849 INFO  [org.jboss.weld.deployer] (MSC service thread 1-2) WFLYWELD0003: Processing weld deployment granular-distributable.war
00:58:31,855 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-2) WFLYEJB0473: JNDI bindings for session bean named 'TopologyChangeListenerBean' in deployment unit 'deployment "granular-distributable.war"' are as follows:

	java:global/granular-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:app/granular-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:module/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:jboss/exported/granular-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:global/granular-distributable/TopologyChangeListenerBean
	java:app/granular-distributable/TopologyChangeListenerBean
	java:module/TopologyChangeListenerBean

00:58:31,879 INFO  [org.jboss.weld.deployer] (MSC service thread 1-2) WFLYWELD0006: Starting Services for CDI deployment: granular-distributable.war
00:58:31,882 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 23) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,882 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 23) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,884 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,884 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,885 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 3) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,886 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 3) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,890 INFO  [org.jboss.weld.deployer] (MSC service thread 1-2) WFLYWELD0009: Starting weld service for deployment granular-distributable.war
00:58:31,893 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,894 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,896 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 23) WFLYCLINF0002: Started dist cache from ejb container
00:58:31,899 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 87) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,899 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 87) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:31,915 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 3) WFLYCLINF0002: Started client-mappings cache from ejb container
00:58:31,915 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 87) WFLYCLINF0002: Started granular-distributable.war cache from web container
00:58:31,918 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 35) WFLYCLINF0002: Started routing cache from web container
00:58:32,104 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 87) WFLYUT0021: Registered web context: /granular-distributable
00:58:32,115 INFO  [org.jboss.as.server] (management-handler-thread - 3) WFLYSRV0010: Deployed "granular-distributable.war" (runtime-name : "granular-distributable.war")
00:58:32,121 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-1
00:58:32,132 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-1\standalone\data\content\6c\00e6e6dc5a810408646389f081dbe9e850b9a2\content
00:58:32,133 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-1) WFLYSRV0027: Starting deployment of "granular-distributable.war" (runtime-name: "granular-distributable.war")
00:58:32,156 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-8) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.
00:58:32,156 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-8) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:58:32,157 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-8) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.infinispan:main") which may be changed or removed in future versions without notice.


org.jboss.as.test.clustering.cluster.web.GranularWebFailoverTestCase(SYNC-tcp).testGracefulSimpleFailover: java.lang.AssertionError: expected:<200> but was:<500>
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:69)
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:58)
	at org.jboss.as.test.clustering.ClusterHttpClientUtil.establishTopology(ClusterHttpClientUtil.java:52)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.establishTopology(ClusteredWebFailoverAbstractCase.java:256)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.testFailover(ClusteredWebFailoverAbstractCase.java:112)
	at org.jboss.as.test.clustering.cluster.web.ClusteredWebFailoverAbstractCase.testGracefulSimpleFailover(ClusteredWebFailoverAbstractCase.java:83)
------- Stdout: -------
00:58:47,532 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:58:47,533 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:58:47,533 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-0
00:58:47,543 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\98\f0d78ea0a5c6458771e05f978fdbf7ab131ea4\content
00:58:47,545 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-1) WFLYSRV0027: Starting deployment of "granular-distributable.war" (runtime-name: "granular-distributable.war")
00:58:47,567 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-3) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.
00:58:47,567 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-3) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:58:47,567 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-3) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.infinispan:main") which may be changed or removed in future versions without notice.
00:58:47,572 INFO  [org.jboss.weld.deployer] (MSC service thread 1-3) WFLYWELD0003: Processing weld deployment granular-distributable.war
00:58:47,576 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-3) WFLYEJB0473: JNDI bindings for session bean named 'TopologyChangeListenerBean' in deployment unit 'deployment "granular-distributable.war"' are as follows:

	java:global/granular-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:app/granular-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:module/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:jboss/exported/granular-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:global/granular-distributable/TopologyChangeListenerBean
	java:app/granular-distributable/TopologyChangeListenerBean
	java:module/TopologyChangeListenerBean

00:58:47,599 INFO  [org.jboss.weld.deployer] (MSC service thread 1-3) WFLYWELD0006: Starting Services for CDI deployment: granular-distributable.war
00:58:47,601 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 19) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,602 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 19) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,604 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,605 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,605 INFO  [org.jboss.weld.deployer] (MSC service thread 1-4) WFLYWELD0009: Starting weld service for deployment granular-distributable.war
00:58:47,606 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,606 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,607 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,607 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 82) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,613 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 19) WFLYCLINF0002: Started dist cache from ejb container
00:58:47,616 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 82) WFLYCLINF0002: Started routing cache from web container
00:58:47,621 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 10) WFLYCLINF0002: Started client-mappings cache from ejb container
00:58:47,624 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,625 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:58:47,631 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 82) WFLYCLINF0002: Started granular-distributable.war cache from web container
00:58:47,758 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 86) WFLYUT0021: Registered web context: /granular-distributable
00:58:47,776 INFO  [org.jboss.as.server] (management-handler-thread - 2) WFLYSRV0010: Deployed "granular-distributable.war" (runtime-name : "granular-distributable.war")
00:58:47,788 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-1
00:58:47,799 INFO  [org.jboss.as.repository] (management-handler-thread - 1) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-1\standalone\data\content\98\f0d78ea0a5c6458771e05f978fdbf7ab131ea4\content
00:58:47,800 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-5) WFLYSRV0027: Starting deployment of "granular-distributable.war" (runtime-name: "granular-distributable.war")
00:58:47,824 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.
00:58:47,824 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:58:47,825 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-2) WFLYSRV0018: Deployment "deployment.granular-distributable.war" is using a private module ("org.infinispan:main") which may be changed or removed in future versions without notice.


```

--- a/undertow/src/main/resources/schema/wildfly-undertow_1_0.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_1_0.xsd
@@ -73,7 +73,7 @@
         <xs:attribute name="enabled" type="xs:boolean" default="true"/>
         <xs:attribute name="max-post-size" type="xs:long" default="10485760"/>
         <xs:attribute name="buffer-pipelined-data" type="xs:boolean" default="true"/>
-        <xs:attribute name="max-header-size" type="xs:long" default="5120"/>
+        <xs:attribute name="max-header-size" type="xs:long" default="1048576"/>
         <xs:attribute name="max-parameters" type="xs:long" default="1000"/>
         <xs:attribute name="max-headers" type="xs:long" default="200"/>
         <xs:attribute name="max-cookies" type="xs:long" default="200"/>

--- a/undertow/src/main/resources/schema/wildfly-undertow_1_0.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_1_0.xsd
@@ -253,7 +253,7 @@
         <xs:attribute name="pattern" use="optional" type="xs:string" default="common"/>
         <xs:attribute name="worker" use="optional" type="xs:string" default="default"/>
         <xs:attribute name="directory" use="optional" type="xs:string" default="${jboss.server.log.dir}"/>
-        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log"/>
+        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log."/>
         <xs:attribute name="rotate" use="optional" type="xs:string" default="true"/>
     </xs:complexType>
     <xs:complexType name="errorPageType">

--- a/undertow/src/main/resources/schema/wildfly-undertow_1_1.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_1_1.xsd
@@ -267,8 +267,8 @@
         <xs:attribute name="pattern" use="optional" type="xs:string" default="common"/>
         <xs:attribute name="worker" use="optional" type="xs:string" default="default"/>
         <xs:attribute name="directory" use="optional" type="xs:string" default="${jboss.server.log.dir}"/>
-        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log"/>
-        <xs:attribute name="suffix" use="optional" type="xs:string" default=".log"/>
+        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log."/>
+        <xs:attribute name="suffix" use="optional" type="xs:string" default="log"/>
         <xs:attribute name="rotate" use="optional" type="xs:string" default="true"/>
     </xs:complexType>
     <xs:complexType name="errorPageType">

--- a/undertow/src/main/resources/schema/wildfly-undertow_1_1.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_1_1.xsd
@@ -82,7 +82,7 @@
                 <xs:attribute name="enabled" type="xs:boolean" default="true"/>
                 <xs:attribute name="max-post-size" type="xs:long" default="10485760"/>
                 <xs:attribute name="buffer-pipelined-data" type="xs:boolean" default="true"/>
-                <xs:attribute name="max-header-size" type="xs:long" default="5120"/>
+                <xs:attribute name="max-header-size" type="xs:long" default="1048576"/>
                 <xs:attribute name="max-parameters" type="xs:long" default="1000"/>
                 <xs:attribute name="max-headers" type="xs:long" default="200"/>
                 <xs:attribute name="max-cookies" type="xs:long" default="200"/>

--- a/undertow/src/main/resources/schema/wildfly-undertow_1_2.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_1_2.xsd
@@ -84,7 +84,7 @@
                 <xs:attribute name="resolve-peer-address" type="xs:boolean" default="false"/>
                 <xs:attribute name="max-post-size" type="xs:long" default="10485760"/>
                 <xs:attribute name="buffer-pipelined-data" type="xs:boolean" default="true"/>
-                <xs:attribute name="max-header-size" type="xs:long" default="5120"/>
+                <xs:attribute name="max-header-size" type="xs:long" default="1048576"/>
                 <xs:attribute name="max-parameters" type="xs:long" default="1000"/>
                 <xs:attribute name="max-headers" type="xs:long" default="200"/>
                 <xs:attribute name="max-cookies" type="xs:long" default="200"/>

--- a/undertow/src/main/resources/schema/wildfly-undertow_1_2.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_1_2.xsd
@@ -276,8 +276,8 @@
         <xs:attribute name="pattern" use="optional" type="xs:string" default="common"/>
         <xs:attribute name="worker" use="optional" type="xs:string" default="default"/>
         <xs:attribute name="directory" use="optional" type="xs:string" default="${jboss.server.log.dir}"/>
-        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log"/>
-        <xs:attribute name="suffix" use="optional" type="xs:string" default=".log"/>
+        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log."/>
+        <xs:attribute name="suffix" use="optional" type="xs:string" default="log"/>
         <xs:attribute name="rotate" use="optional" type="xs:string" default="true"/>
     </xs:complexType>
     <xs:complexType name="errorPageType">

--- a/undertow/src/main/resources/schema/wildfly-undertow_2_0.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_2_0.xsd
@@ -293,8 +293,8 @@
         <xs:attribute name="worker" use="optional" type="xs:string" default="default"/>
         <xs:attribute name="directory" use="optional" type="xs:string" default="${jboss.server.log.dir}"/>
         <xs:attribute name="relative-to" use="optional" type="xs:string" />
-        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log"/>
-        <xs:attribute name="suffix" use="optional" type="xs:string" default=".log"/>
+        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log."/>
+        <xs:attribute name="suffix" use="optional" type="xs:string" default="log"/>
         <xs:attribute name="rotate" use="optional" type="xs:string" default="true"/>
         <xs:attribute name="use-server-log" use="optional" type="xs:string" default="false"/>
     </xs:complexType>

--- a/undertow/src/main/resources/schema/wildfly-undertow_2_0.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_2_0.xsd
@@ -89,7 +89,7 @@
                 <xs:attribute name="resolve-peer-address" type="xs:boolean" default="false"/>
                 <xs:attribute name="max-post-size" type="xs:long" default="10485760"/>
                 <xs:attribute name="buffer-pipelined-data" type="xs:boolean" default="true"/>
-                <xs:attribute name="max-header-size" type="xs:long" default="5120"/>
+                <xs:attribute name="max-header-size" type="xs:long" default="1048576"/>
                 <xs:attribute name="max-parameters" type="xs:long" default="1000"/>
                 <xs:attribute name="max-headers" type="xs:long" default="200"/>
                 <xs:attribute name="max-cookies" type="xs:long" default="200"/>

--- a/undertow/src/main/resources/schema/wildfly-undertow_3_0.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_3_0.xsd
@@ -90,7 +90,7 @@
                 <xs:attribute name="resolve-peer-address" type="xs:boolean" default="false"/>
                 <xs:attribute name="max-post-size" type="xs:long" default="10485760"/>
                 <xs:attribute name="buffer-pipelined-data" type="xs:boolean" default="true"/>
-                <xs:attribute name="max-header-size" type="xs:long" default="5120"/>
+                <xs:attribute name="max-header-size" type="xs:long" default="1048576"/>
                 <xs:attribute name="max-parameters" type="xs:long" default="1000"/>
                 <xs:attribute name="max-headers" type="xs:long" default="200"/>
                 <xs:attribute name="max-cookies" type="xs:long" default="200"/>

--- a/undertow/src/main/resources/schema/wildfly-undertow_3_0.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_3_0.xsd
@@ -344,8 +344,8 @@
         <xs:attribute name="worker" use="optional" type="xs:string" default="default"/>
         <xs:attribute name="directory" use="optional" type="xs:string" default="${jboss.server.log.dir}"/>
         <xs:attribute name="relative-to" use="optional" type="xs:string" />
-        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log"/>
-        <xs:attribute name="suffix" use="optional" type="xs:string" default=".log"/>
+        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log."/>
+        <xs:attribute name="suffix" use="optional" type="xs:string" default="log"/>
         <xs:attribute name="rotate" use="optional" type="xs:string" default="true"/>
         <xs:attribute name="use-server-log" use="optional" type="xs:string" default="false"/>
         <xs:attribute name="extended" use="optional" type="xs:string" default="false" />

--- a/undertow/src/main/resources/schema/wildfly-undertow_3_1.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_3_1.xsd
@@ -90,7 +90,7 @@
                 <xs:attribute name="resolve-peer-address" type="xs:boolean" default="false"/>
                 <xs:attribute name="max-post-size" type="xs:long" default="10485760"/>
                 <xs:attribute name="buffer-pipelined-data" type="xs:boolean" default="false"/>
-                <xs:attribute name="max-header-size" type="xs:long" default="5120"/>
+                <xs:attribute name="max-header-size" type="xs:long" default="1048576"/>
                 <xs:attribute name="max-parameters" type="xs:long" default="1000"/>
                 <xs:attribute name="max-headers" type="xs:long" default="200"/>
                 <xs:attribute name="max-cookies" type="xs:long" default="200"/>

--- a/undertow/src/main/resources/schema/wildfly-undertow_3_1.xsd
+++ b/undertow/src/main/resources/schema/wildfly-undertow_3_1.xsd
@@ -357,8 +357,8 @@
         <xs:attribute name="worker" use="optional" type="xs:string" default="default"/>
         <xs:attribute name="directory" use="optional" type="xs:string" default="${jboss.server.log.dir}"/>
         <xs:attribute name="relative-to" use="optional" type="xs:string" />
-        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log"/>
-        <xs:attribute name="suffix" use="optional" type="xs:string" default=".log"/>
+        <xs:attribute name="prefix" use="optional" type="xs:string" default="access_log."/>
+        <xs:attribute name="suffix" use="optional" type="xs:string" default="log"/>
         <xs:attribute name="rotate" use="optional" type="xs:string" default="true"/>
         <xs:attribute name="use-server-log" use="optional" type="xs:string" default="false"/>
         <xs:attribute name="extended" use="optional" type="xs:string" default="false" />

